### PR TITLE
feat: giveaways

### LINKED ISSUE #58: giveaway
- [x] Store Keys
- [x] Check if user got a key already
- [x] Check if user is steam owner

### PR DESCRIPTION
**Pull request by WilsontheWolf**
Store keys.
Firebase import now imports keys.
Allow users to redeem keys if they won a giveaway.

Closes #58

### CODE REVIEW

File: src/dto/KeyDto.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/** Class representing key data. */
+export default class KeyDto {
+    /**
+     * Create new key data.
+     * @param {String} value - The key's value.
+     */
+    constructor(value = '') {
+        this.value = value;
```
Comment (KommentatorForAll) [inline src/dto/KeyDto.js:8]: Why do we need a whole class if there's only one value inside?

**Review by KommentatorForAll — CHANGES_REQUESTED** [review]

File: src/dto/KeyDto.js (hunk 1 above)
Comment (WilsontheWolf) [reply src/dto/KeyDto.js:8]: uhh idk consistency?

File: src/helper/main.js [hunk 2]
```diff
@@ -139,6 +142,11 @@ import { logAndDie, helpMessage, databases } from './helperUtil';
                 for (const [id, response] of newAutoResponces) {
                     if (id !== 'holder') responseDb.set(id, response);
                 }
+
+                console.log('Storing key data...');
+                for (const key of newKeys) {
+                    if (key.value !== 'holder') keyDb.set(key.value, key);
```
Comment (Chicken) [inline src/helper/main.js:148]: Wait... so the keydb is structured as `'key' => { value: 'key' }` ???
I know that is probably the best solution but damn... document based dabases aint the best for this :D
I'll approve it anyway.

**Review by Chicken — APPROVED** [review]

File: src/dto/KeyDto.js (hunk 1 above)
Comment (WilsontheWolf) [reply src/dto/KeyDto.js:8]: idk your the database master what should I do?

**Review by KommentatorForAll — APPROVED** [review]
i guess?

**Pull request merged**